src/Stratis.Bitcoin/Base/InitialBlockDownloadState.cs

```diff
@@ -65,7 +65,8 @@ public bool IsInitialBlockDownload()
                 return true;
 
             this.logger.LogTrace("BlockTimeUnixSeconds={0}, DateTimeProviderTime={1}, ConsensusSettingsMaxTipAge={2}",
-                this.chainState.ConsensusTip.Header.BlockTime.ToUnixTimeSeconds(), this.chainState.ConsensusTip.Header.BlockTime.ToUnixTimeSeconds(),
+                this.chainState.ConsensusTip.Header.BlockTime.ToUnixTimeSeconds(),
```

Comment:
For the sake of precision this fix should probably first acquire the values form the components then log (and make the comparison).